retrofit/src/main/java/retrofit/RequestBuilder.java

```diff
@@ -240,87 +266,90 @@ void setArguments(Object[] args) {
-      //      if (entryKey == null) {
-      //        throw new IllegalArgumentException(
-      //            "Parameter #" + (i + 1) + " part map contained null key.");
-      //      }
-      //      String entryName = entryKey.toString();
-      //      Object entryValue = entry.getValue();
-      //      if (entryValue != null) { // Skip null values.
-      //        if (entryValue instanceof TypedOutput) {
-      //          multipartBody.addPart(entryName, transferEncoding, (TypedOutput) entryValue);
-      //        } else if (entryValue instanceof String) {
-      //          multipartBody.addPart(entryName, transferEncoding,
-      //              new TypedString((String) entryValue));
-      //        } else {
-      //          multipartBody.addPart(entryName, transferEncoding,
-      //              converter.toBody(entryValue, entryValue.getClass()));
-      //        }
-      //      }
-      //    }
-      //  }
+//      } else if (annotationType == Field.class) {
+//        if (value != null) { // Skip null values.
+//          Field field = (Field) annotation;
+//          String name = field.value();
+//          boolean encodeName = field.encodeName();
+//          boolean encodeValue = field.encodeValue();
+//          if (value instanceof Iterable) {
+//            for (Object iterableValue : (Iterable<?>) value) {
+//              if (iterableValue != null) { // Skip null values.
+//                formBody.addField(name, encodeName, iterableValue.toString(), encodeValue);
+//              }
+//            }
+//          } else if (value.getClass().isArray()) {
+//            for (int x = 0, arrayLength = Array.getLength(value); x < arrayLength; x++) {
+//              Object arrayValue = Array.get(value, x);
+//              if (arrayValue != null) { // Skip null values.
+//                formBody.addField(name, encodeName, arrayValue.toString(), encodeValue);
+//              }
+//            }
+//          } else {
+//            formBody.addField(name, encodeName, value.toString(), encodeValue);
+//          }
+//        }
+//      } else if (annotationType == FieldMap.class) {
+//        if (value != null) { // Skip null values.
+//          FieldMap fieldMap = (FieldMap) annotation;
+//          boolean encodeNames = fieldMap.encodeNames();
+//          boolean encodeValues = fieldMap.encodeValues();
+//          for (Map.Entry<?, ?> entry : ((Map<?, ?>) value).entrySet()) {
+//            Object entryKey = entry.getKey();
+//            if (entryKey == null) {
+//              throw new IllegalArgumentException(
+//                  "Parameter #" + (i + 1) + " field map contained null key.");
+//            }
+//            Object entryValue = entry.getValue();
+//            if (entryValue != null) { // Skip null values.
+//              formBody.addField(entryKey.toString(), encodeNames, entryValue.toString(),
+//                  encodeValues);
+//            }
+//          }
+//        }
```

Comment:
Can you revert the comment changes here? It adds needless noise to the commit which messes with history and prevent cherry-picking and reverting (not that I expect those to need to happen).